zips/draft-ecc-community-and-coinholder.md

```diff
@@ -0,0 +1,188 @@
+
+This output MUST follow the same consensus rules as if there were an additional ``FS_COINHOLDER_ONETIME`` funding stream lasting for a single block, with value for that block equal to the amount in the Deferred Dev Fund Lockbox at the activation height. (This value can be predicted in advance, given the activation height.) At the same time, the balance of the Deferred Dev Fund Lockbox will be reduced by that amount, i.e. reset to zero.
+
+Exactly one of the options in the next three sections will also be taken.
+
+### Option 1: Extend the lockbox funding stream
+
+This option extends the ``FS_DEFERRED`` lockbox funding stream by a further 1,260,000 blocks (approximately 3 years), ending at the 3rd halving.
+
+### Option 2: Additional funding stream
+
+An additional funding stream will be established, consisting of 12% of the block subsidy paid to a 3-of-5 P2SH multisig with keys held by the same Key-holder Organizations as above. The Coin-holder-controlled Fund is considered to include both funds from the ``FS_COINHOLDER`` funding stream, and funds from the one-time lockbox disbursal described above.
+
+This stream will replace the existing ``FS_DEFERRED`` lockbox funding stream [^zip-1015-funding-streams]. That is, the end height of ``FS_DEFERRED`` will be changed to the activation height of this ZIP. The new ``FS_COINHOLDER`` stream will start at the same height and last until the 3rd halving.
+
+### Option 3: Periodic lockbox disbursement
+
+This would require a mechanism to pay from the Deferred Dev Fund Lockbox to a 3-of-5 P2SH multisig with keys held by the same Key-holder Organizations as above, at periodic intervals. [TBD schedule and period.]
+
+### Requirements on use of the Coin-holder-controlled Fund
+
+The Key-Holder Organizations would be bound by a legal agreement to only use funds held in the Coin-holder-controlled Fund according to the specifications in this ZIP, expressed in suitable legal language. In particular, all requirements on the use of Deferred Dev Fund Lockbox funds in ZIP 1015 [^zip-1015-lockbox] MUST be followed for the funds obtained from the one-time disbursal.
+
+The Financial Privacy Foundation (FPF) would administer the Coin-holder-controlled Fund based on decisions taken by coin-holder voting, following a model decided by the community and specified in another ZIP [TBD].
+
+### Disbursement process
+
+1. Anyone can submit a grant application via a process agreed between the FPF, the Zcash Foundation, and the Electric Coin Company.
+2. Grant applications with total below USD 500,000 (or equivalent in another currency) would be directed to ZCG.
+3. For grant applications above this threshold, a 30-day community review and feedback period would start.
+4. If the grant applications should proceed to a coin-holder vote according to the agreed process, then coin-holders would be asked to vote on it.
```

Comment:
```suggestion
4. If a grant application proceeds to a coin-holder vote according to the agreed process, then coin-holders will be asked to vote on it.
```
This doesn't specify a timeline; does the vote take place at the end of the 30 day window? As written, this looks like it allows a vote to be deferred indefinitely.

Reply:
Also: at present this doesn't define a coin-holder voting mechanism. Using Hanh's coin voting mechanism restricts votes to funds held in the Orchard pool; if that's intended, then it should be explicitly specified; however, that might be construed to contradict https://github.com/zcash/zips/pull/989/files#diff-8528553152727684ca93a225882ceed8e57a309ae833f5f517f0acbf573cd61aR129 which says that organizations cannot be restricted from voting their coins; if some barrier (such as lack of server-grade HSM support for Orchard) prevents an organization from safely voting their coins, is that a problem?

Reply:
> This doesn't specify a timeline; does the vote take place at the end of the 30 day window?

As currently written, the "process agreed upon by the Key-Holder Organizations" decides that. So it's not ambiguous, but we might want to tighten it up.